spreadsheet/formula/cellref.go

```diff
@@ -26,3 +28,31 @@ func (c CellRef) Eval(ctx Context, ev Evaluator) Result {
 func (c CellRef) Reference(ctx Context, ev Evaluator) Reference {
 	return Reference{Type: ReferenceTypeCell, Value: c.s}
 }
+
+// String returns a string representation of CellRef.
+func (c CellRef) String() string {
+	return c.s
+}
+
+// MoveLeft makes a reference to point to a cell which one step left from the original one after removing a column.
+func (c CellRef) MoveLeft(q *MoveQuery) Expression {
+	if q.MoveCurrentSheet {
+		c.s = moveCellLeft(c.s, q.ColumnIdx)
+	}
+	return c
+}
+
+func moveCellLeft(refStr string, columnIdxToRemove uint32) string {
```

Comment:
please add some comment, why does moving cell left return a string?
And why is it returning the String() method on the expression?  

What does the String() method actually return?  A value of a cell in string readable format, or something else?